packages/bible-book-names-intl/src/data/translations/de.json

```diff
@@ -0,0 +1,578 @@
+      "Na",
+      "Nah"
+    ],
+    "startNumber": 0
+  },
+  "35": {
+    "name": "Habakuk",
+    "shortNames": [
+      "Hb",
+      "Hab",
+      "Hk",
+      "Habk"
+    ],
+    "startNumber": 0
+  },
+  "36": {
+    "name": "Zephanja",
+    "shortNames": [
+      "Zp",
+      "Zep",
+      "Zeph",
+      "Ze",
+      "Zefanja",
+      "Zef"
+    ],
+    "startNumber": 0
+  },
+  "37": {
+    "name": "Haggai",
+    "shortNames": [
+      "Ha",
+      "Hag",
+      "Hagg"
+    ],
+    "startNumber": 0
+  },
+  "38": {
+    "name": "Sacharja",
+    "shortNames": [
+      "Sach"
+    ],
+    "startNumber": 0
+  },
+  "39": {
+    "name": "Maleachi",
+    "shortNames": [
+      "Ml",
+      "Mal",
+      "Mlc"
+    ],
+    "startNumber": 0
+  },
+  "40": {
+    "name": "Matthäus",
+    "shortNames": [
+      "Mt",
+      "Matt",
+      "Mat"
+    ],
+    "startNumber": 0
+  },
+  "41": {
+    "name": "Markus",
+    "shortNames": [
+      "Mk",
+      "Mrk"
+    ],
+    "startNumber": 0
+  },
+  "42": {
+    "name": "Lukas",
+    "shortNames": [
+      "Lk",
+      "Luk",
+      "Lu"
+    ],
+    "startNumber": 0
+  },
+  "43": {
+    "name": "Johannes",
+    "shortNames": [
+      "Jn",
+      "Joh",
+      "Jo"
+    ],
+    "startNumber": 0
+  },
+  "44": {
+    "name": "Apostelgeschichte",
+    "shortNames": [
+      "Ac",
+      "Act",
+      "Apg"
+    ],
+    "startNumber": 0
+  },
+  "45": {
+    "name": "Römer",
+    "shortNames": [
+      "Rö",
+      "Röm"
+    ],
+    "startNumber": 0
+  },
+  "46": {
+    "name": "Korinther",
+    "shortNames": [
+      "Ko",
+      "Kor"
+    ],
+    "startNumber": 1
+  },
+  "47": {
+    "name": "Korinther",
+    "shortNames": [
+      "Ko",
+      "Kor"
+    ],
+    "startNumber": 2
+  },
+  "48": {
+    "name": "Galater",
+    "shortNames": [
+      "Ga",
+      "Gal"
+    ],
+    "startNumber": 0
+  },
+  "49": {
+    "name": "Epheser",
+    "shortNames": [
+      "Ep",
+      "Eph"
+    ],
+    "startNumber": 0
+  },
+  "50": {
+    "name": "Philipper",
+    "shortNames": [
+      "Phi",
+      "Phil"
+    ],
+    "startNumber": 0
+  },
+  "51": {
+    "name": "Kolosser",
+    "shortNames": [
+      "Kol"
+    ],
+    "startNumber": 0
+  },
+  "52": {
+    "name": "Thessalonicher",
+    "shortNames": [
+      "Th",
+      "Thess",
+      "Thes"
+    ],
+    "startNumber": 1
+  },
+  "53": {
+    "name": "Thessalonicher",
+    "shortNames": [
+      "Th",
+      "Thess",
+      "Thes"
+    ],
+    "startNumber": 2
+  },
+  "54": {
+    "name": "Timotheus",
+    "shortNames": [
+      "Ti",
+      "Tim"
+    ],
+    "startNumber": 1
+  },
+  "55": {
+    "name": "Timotheus",
+    "shortNames": [
+      "Ti",
+      "Tim"
+    ],
+    "startNumber": 2
+  },
+  "56": {
+    "name": "Titus",
+    "shortNames": [
+      "Ti",
+      "Tit",
+      "Tt",
+      "Ts"
+    ],
+    "startNumber": 0
+  },
+  "57": {
+    "name": "Philemon",
+    "shortNames": [
+      "Pm",
+      "Phil",
```

Comment:
```
src/tests/translations.test.ts:
  ✓ test translation structure > test english [2.27ms]
  ✓ test all translations > there are at least one valid JSON file [0.03ms]
  ✓ test all translations > test each translation [1.23ms]
  Phil is duplicated among other books
  
  Error: Expected: false
  Received: true
  ```
  
  I think Line 433 and Line 492, the short names are the same. Which cause the testing failing

Reply:
You're correct. Fixed it.